packages/fxa-settings/src/pages/Index/container.tsx

```diff
@@ -113,7 +130,7 @@ export const IndexContainer = ({
       try {
         if (!isEmailValid(email)) {
           return {
-            error: getErrorWithDefinedErrNo(AuthUiErrors.EMAIL_REQUIRED),
+            error: AuthUiErrors.EMAIL_REQUIRED as AuthError,
```

Comment:
There's an issue with types here maybe there's a better way to handle, need to look again

Reply:
Yup, seems like issues with AuthError vs AuthUIError. I'd maybe circle back on this in separate PR. As I looked into it theres a slew of types that need to be reconciled out. 

I think see a cleaner way to do this if you want me to take shot at it...